slides/developer-guide/working-with-notes-slides/delete-notes-slides/_index.md

```diff
@@ -0,0 +1,488 @@
+---
+title: "Delete Notes Slides"
+type: docs
+url: /delete-notes-slides/
+weight: 35
+---
+
+## **Introduction**
+
+When you add speaker notes to a slide, a Notes slide is created in your PowerPoint presentation. The Notes slide can contain text, shapes, etc. The following method allows you to delete the Notes slide associated with a regular slide.
+
+![Notes slide](notes_slide.png)
+
+{{% alert color="primary" %}} 
+When you delete a non-existent Notes slide, you will receive an error message: "Object reference not set to an instance of an object.". You can use the [NotesSlideExists](https://reference.aspose.cloud/slides/#/NotesSlide/NotesSlideExists) or [NotesSlideExistsOnline](https://reference.aspose.cloud/slides/#/NotesSlide/NotesSlideExistsOnline) methods to check if a Notes slide exists.
```

Comment:
1. I would rather say "When you **attempt** to delete a non-existent Notes slide"

2. Please do not mention "Object reference not set..." error. It is an inappropriate error message. Please create a task to fix that. I think it is sufficient just to say "you will get an error."